### DISCUSSION TITLE
Remove water dots

### DISCUSSION THREAD

**Discussion by esysfound** [organicmaps/organicmaps#7164] [Ideas]
At certain zoom scales the blue watter dots really get in the way. In cities at least can these be turned off or folded into a terrain overlay? Thx

**Comment by hb0nd** [comment]
can you share an example location?

**Reply by biodranik** [reply]
Zürich? I think visibility of water can be reduced in the general style after introducing the outdoors style.

**Comment by esysfound** [comment]
It is also siedlings - and water wells. All over e.g. Berlin, Germany. Just find them all arbitrary and very distracting when I just want to concentrate on route finding and streets.
Fine that the info is there - just think it should be layered and not visible in default view.

Sent with [Proton Mail](https://proton.me/) secure email.

**Reply by patepelo** [reply]
@organicmaps/styles

**Comment by esysfound** [comment]
Link not working.

Can't you also just write in plain English what solution this offers? Sometimes the engineering chic is really just absurd. I spent 10 min trying to get your link to work, pairing back to roots, etc. Easy, quick for you - for me, just the opposite.

What would it cost you - 5 seconds? - to explain properly.

Thx

Sent with [Proton Mail](https://proton.me/) secure email.

**Reply by patepelo** [reply]
Sorry for the misunderstanding. With that input I'm tagging (ping) the team in charge of styles to gather the attention to this discussion. There are so many things going on many might not have read this thread yet.

On the issue with the water. It's very subjective and there's no one size fits all. But I believe it's not the first time this has been suggested. It's important to see other open issues and try to keep it in one. 

Let's hear some other voices and see if we can adjust.

Me personally, I like it the way it's now. I guess it's a first world problem, the privilege of having drinking fountains all over our cities. 😊

Since the new outdoors layer gives more exposure to water, it could be adjusted in the main layer.

**Comment by esysfound** [comment]
Thanks we will see

Sent with [Proton Mail](https://proton.me/) secure email.

**Comment by pastk** [comment]
Did you check it in the latest Feb release already?
Water sources were made to show up one zoom level later already (in https://github.com/organicmaps/organicmaps/pull/7056).

Now we're discussing whether it makes sense to low them one zoom level more or leave as it is, please see the discussion in https://github.com/organicmaps/organicmaps/pull/7192

**Comment by esysfound** [comment]
thanks

Sent with [Proton Mail](https://proton.me/) secure email.

**Comment by esysfound** [comment]
It is a little better - but I think it could be lowered more, still. I understand some cities/places are water stressed. But for most general viewing, this is really not an issue. Maybe even less so than say toilets.....
I don't know how the zoom number scale works, but water should be really visible only at like 100 meters, IMHO.

Thanks for working on this

Sent with [Proton Mail](https://proton.me/) secure email.

**Comment by euf** [comment]
A few screenshots for the different zooms on the problematic area would be helpful to understand the issue! ❤️
